.babelrc

```diff
@@ -4,6 +4,7 @@
   ],
 
   "plugins": [
+    "array-includes",
```

Comment:
Well, I've tried several ways to solve the issue:
* [core-js](https://www.npmjs.com/package/core-js) - it's a huge collection of different shims/polyfills, it's great when you need to shim a lot of stuff or you're not in a browser - bundle that it generates just for array.includes is unreasonably big :/
* [array-includes](https://www.npmjs.com/package/array-includes) - a bit better, but requires additional gulp task to compile browser-compatible bundle - didn't even try to write it, as it feels too much just for this shim :)
* And this one - just replace `includes` with babel plugin - easiest way and don't affect peformance in any way.

And the main factor is that Edge array.includes implementation is already at the [Preview stage](https://developer.microsoft.com/en-us/microsoft-edge/platform/status/arrayprototypeincludeses2016)! So we can get rid of this plugin soon enough.